jwst/background/tests/test_background.py

```diff
@@ -134,3 +146,162 @@ def test_nirspec_gwa_ytitl(_jail, background, science_image):
     assert result.meta.cal_step.back_sub == 'SKIPPED'
 
     back_image.close()
+
+@pytest.fixture(scope='module')
+def make_wfss_datamodel():
+    """Generate WFSS Observation"""
+    wcsinfo = {
+        'dec_ref': -27.79156387419731,
+        'ra_ref': 53.16247756038121,
+        'roll_ref': 0.04254766236781744,
+        'v2_ref': -290.1,
+        'v3_ref': -697.5,
+        'v3yangle': 0.56987,
+        'vparity': -1}
+
+    observation = {
+        'date': '2016-09-05',
+        'time': '8:59:37'}
+
+    exposure = {
+        'duration': 11.805952,
+        'end_time': 58119.85416,
+        'exposure_time': 11.776,
+        'frame_time': 0.11776,
+        'group_time': 0.11776,
+        'groupgap': 0,
+        'integration_time': 11.776,
+        'nframes': 1,
+        'ngroups': 8,
+        'nints': 1,
+        'nresets_between_ints': 0,
+        'nsamples': 1,
+        'sample_time': 10.0,
+        'start_time': 58668.72509857639,
+        'zero_frame': False}
+
+    subarray = {'xsize':2048,
```

Comment:
If there's anything that actually uses the subarray information, you'll need `xstart` and `ystart` (both = 1) as well.

Reply:
gotcha, will make the change